src/js/components/DSLFormDropdownPanel.js

```diff
@@ -83,7 +84,7 @@ class DSLFormDropdownPanel extends React.Component {
         </div>
         <div className="panel-cell panel-cell-short flush-top text-align-right">
           <a
-            className="button button-small button-primary-link"
+            className="button button-small button-primary-link flush-right"
```

Comment:
why is the `flush-right` class here and the "Apply" button still there?

Reply:
This was a recommendation from the design team to decrease the paddings in the dropdown modal.

Reply:
Also, it was decided by the design team to keep the apply button although it is automatically updated

Reply:
This is like a "close door" button in an elevator. Sometimes a user needs something obvious to click. Otherwise they have to guess to click outside of the dropdown panel to proceed. @cujojp had strong opinions about this

Reply:
Okay, makes sense 👍